.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+AllCops:
+  TargetRubyVersion: 2.7
+  NewCops: enable
+  Exclude:
+    - 'lib/generators/**/*'
+    - 'tmp/**/*'
+    - 'spec/**/*'
+    - 'vendor/**/*' # added by github actions
+
+Style/Documentation:
+  Enabled: false
+
+Style/SafeNavigation:
+  Enabled: false
+
+Layout/LineLength:
+  Max: 175
```

Comment:
175 is quite long. How about 100?

Reply:
I tried both 100 and 115, but there are quite a few violations that might extend the scope of what is already a very large PR. Keeping it at 125 for now.